proto/auction/v1/query.proto

```diff
@@ -85,8 +88,14 @@ message QueryBidResponse {
 
 message QueryBidsByAuctionRequest {
   uint32 auction_id = 1;
+  Pagination pagination = 2;
 }
 
 message QueryBidsByAuctionResponse {
   repeated Bid bids = 1;
+  uint64 next_page = 2;
+}
+
+message Pagination {
+  uint64 offset = 1;
 }
```

Comment:
yeah I actually saw this and thought it was a bit too much lol E.g. providing the ability to set a custom limit kind of defeats the purpose of pagination imo. i.e. if we want to enforce pagination due to physical memory size of an item then that shouldnt be overridable by the client.

Some may disagree, but it also doesnt really seem to make sense to allow variable limits sizes for various pagination requests as they would return potentially unexpected results. For example, if I have a limit size of 100 on page 1, then another limit size of only 10 on page 2, as a client I'd have to update my key or offset to account for that difference. While we can certainly support this kind of flow, it seems to get pretty complex for a client and rather untraditional as far as more typical pagination goes. 

i guess there's really 2 schools of thought here, max customizability for users, or optimizing for simplicity/straightforwardness. I think the latter camp might actually make for a nicer user experience so clients dont have to think about potentially configuring 5 different fields rather than just 1.

Reply:
I think the most important concern here is just that pagination is consistent with what external clients to Cosmos chains expect. This minimizes the probability we break compatibility.

Reply:
Updated to use cosmos pagination convention.